miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/WebConfig.java

```diff
@@ -44,4 +53,18 @@ public RequestMappingHandlerAdapter requestMappingHandlerAdapter() {
     adapter.setWebBindingInitializer(bindingInitializer());
     return adapter;
   }
+
+  @Autowired
+  private List<BarcodableService<?>> barcodableServices;
+
+  @Bean
+  public Map<EntityType, BarcodableService<?>> barcodableServicesMap() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)